plugins/ipam/static/main.go

```diff
@@ -72,7 +72,9 @@ func canonicalizeIP(ip *net.IP) error {
 	return fmt.Errorf("IP %s not v4 nor v6", *ip)
 }
 
-// NewIPAMConfig creates a NetworkConfig from the given network name.
+// LoadIPAMConfig creates IPAMConfig using json encoded configuration provided
+// as `bytes`. At the moment values provided in envArgs are ignored so there
+// is no possibility to overload by it configuration from json.
```

Comment:
not quite sure on the English here - did you mean "no possibility to overload the json configuration using envArgs" ?

Reply:
Yes, you are correct. Will fix that tomorrow morning.